doc/lrs_options-to-html.py

```diff
@@ -0,0 +1,144 @@
+def add_style(option, val):
```

Comment:
Is it possible to add some check in the GHA static analysis checks that we can build this file?
Otherwise if the user will add something to lrs_option.cmake that this script doesn't like we will find it in the actual release, right?

Reply:
IMO we will not find it at all until we notice it's missing in the generated html.

I can add a condition in the script that will throw an error if it reaches a non-empty line, and then GHA can fail if it sees such line... what do you think?

Reply:
Empty lines should be allowed.
If you would have created a MD instead of an html file, you could of compare it to a golden MD you keep, this way if someone touch the options he should generate a file, verify it looks good and upload it.

With HTML I am not sure it's possible.
When will the script fail?
What kind of checks can you think of that will prevent an issue on actual release?